src/lib/edje/edje_lua2.c

```diff
@@ -3,6 +3,10 @@
 #include "edje_private.h"
 #include <ctype.h>
 
+#ifndef _MSC_VER
```

Comment:
Same as in this [one](https://github.com/expertisesolutions/efl/pull/159/files#r449040104)